packages/botonic-plugin-contentful/tests/nlp/normalizer.test.ts

```diff
@@ -9,6 +11,28 @@ import {
   Word,
 } from '../../src/nlp'
 
+test.each<any>([
+  //acronyms
+  ['o.k.', 'ok'],
+  ['o.k', 'ok'],
+  ['o.k ', 'ok '],
+  ['o.k ', 'ok '],
+  ['U.S.A', 'USA'],
+  ['o.k, vale', 'ok, vale'],
+  //not acronyms
+  ['.a.', '.a.'],
+  ['U..A', 'U..A'],
+  ['a. opcion', 'a. opcion'],
+  ['no separators', 'no separators'],
+  ['final dot.', 'final dot.'],
```

Comment:
What about receiving a text like this: `'textwith.dotattached'`? My feeling is that the preprocess will detect it as an acronym when it shouldn't. 
If this is detected as an acronym, if one of the 2 words is a keyword, it won't be detected as such because the dot will be removed and both words will be unified, right?

Reply:
this is a good testcase, thanks!
I added it and it doesn't recognize as an acronym because we discard the word when more than 1 letter appear at any chunk